eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -0,0 +1,31 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
```

Comment:
This script is called SetTestePipelineVersion.ps1 but it really is specific to the template package. Not a big deal but it seems a little miss-leading. When you come back around to clean this up a little we should consider a way to share the common parts of this in eng/common and then call the language specific version incrementing and changelog update.